src/TiledMap.cs

```diff
@@ -707,8 +707,8 @@ public TiledSourceRect GetSourceRect(TiledMapTileset mapTileset, TiledTileset ti
                 if (i == gid - mapTileset.firstgid)
                 {
                     var result = new TiledSourceRect();
-                    result.x = tileHor * tileset.TileWidth;
-                    result.y = tileVert * tileset.TileHeight;
+                    result.x = tileHor * tileset.TileWidth + tileset.Spacing * tileHor;

```

Comment:
Could just put parathesis around (tileset.TileWidth + tileset.Spacing) so that only one multiply is performed vs 2.  This would before both of these lines (but height for the second obviously)

Reply:
Hey, I was not aware others could review too. Was a bit surprised about that to be honest. But thanks for taking your time to take a look at the PR!

Could you elaborate on why the parentheses need to be added? Did you got some incorrect results? As you may have noticed I have not spent much time in this repository due to my focus shifted to other projects so it has been a while since I took a look at this PR.

Reply:
Technically they don't "need" to be added.  It was just a suggestion for (premature optimization) I suppose.

But my comment on the ticket says that PR #94 actually has the same fix you have here, but also has a fix for Margin, so it should be probably be used instead of your fix since it fixes 2 different things.

I didn't mean to indicate your suggestion was "wrong", just wanted to see if I could help spark a conversation that gets some of these PR's merged in.

Reply:
Well, I am not the author of this PR so it was not my suggestion. But I agree it could improve readability. Mathematically it works as it should but I get where you're coming from. Will have a look at both PR and compare them.

Reply:
Oh my, that's what I get for replying while watching Hulu. Sorry i thought you were the requestor.  Haha yes I also thought it was odd I could review things on your repository.

Reply:
No prob, I understand. That said, I find it a little odd to be a default setting that anyone can review. I changed something in the settings just now. Curious if that fixed it. Could you try and see if you can actually review another PR?

Reply:
![image](https://user-images.githubusercontent.com/4995472/229452862-666441f0-38b8-4fd5-b260-0787c2d3cfd5.png)

Yep it shows me and let's me review, I event went to a different pr.

Reply:
Yeah this article makes it sound like it's open to public??? Weird? Now I gotta go check my public repos lol

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-pull-request-reviews-in-your-repository

Reply:
Perhaps I can approve but I can't merge?

Reply:
Yea I found the same page. Already consulted the GitHub community on Reddit cause I am a bit at loss here now. I also can only draw the same conclusion that apparently anyone can fuck with your pull requests when your repo is public. Not really a desired situation imho. 

I wish to keep control over my code and it feels like I barely have control over how anyone can interact with my repository. First the branching regulations and now this crap. lol

Let's rest it for now. I am glad I figured it out this way rather than the nasty way. I will wait out the response on Reddit and see if I can contact GitHub support to ask the same question. Other than that, I will have to deal with the consequences as long as this repo is public I guess. Thank you for taking your time by the way, in all hurries forgot to even properly say thanks. So hereby, thanks!

Reply:
Yeah it appears github at some point (perhaps always?) made public repos open.

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews

You'd have to lock the branch down with a branch protection option in settings.

Settings > code and automation > branches
Then down under branch protection click add rule and see if those settings work.

![image](https://user-images.githubusercontent.com/4995472/229459584-dc1bcd7c-4e8e-43bb-aa03-12d8cf39f270.png)